packages/design-system/src/components/slider/slider.tsx

```diff
@@ -0,0 +1,311 @@
+import {
+  createContext,
+  forwardRef,
+  useCallback,
+  useMemo,
+  type ForwardedRef,
+} from 'react';
+import {
+  Provider,
+  Slider as RACSlider,
+  SliderOutput as RACSliderOutput,
+  SliderThumb as RACSliderThumb,
+  SliderTrack as RACSliderTrack,
+  LabelContext,
+  type ContextValue,
+  type LabelProps,
+  type TextProps,
+  DEFAULT_SLOT,
+} from 'react-aria-components';
+import {
+  AriaLabelContext,
+  AriaTextContext,
+  InputContext,
+  type InputProps,
+} from '../../components';
+import { useContextProps, useDefaultProps, useTheme } from '../../hooks';
+import type {
+  SliderProps,
+  SliderRenderProps,
+  SliderThumbProps,
+  SliderThumbRenderProps,
+  SliderTrackProps,
+  SliderTrackRenderProps,
+  SliderOutputProps,
+} from './types';
+import { callRenderProps, inlineVars, mergeClassNames } from '../../utils';
+import {
+  sliderClassNames,
+  sliderThumbStateVars,
+  sliderTrackStateVars,
+  sliderStateVars,
+} from './slider.css';
+
+export const SliderOutputContext =
+  createContext<ContextValue<SliderOutputProps, HTMLOutputElement>>(null);
+
+export const SliderOutput = forwardRef(function SliderOutput(
+  props: SliderOutputProps,
+  ref: ForwardedRef<HTMLOutputElement>,
+) {
+  [props, ref] = useContextProps(props, ref, SliderOutputContext);
+
+  props = useDefaultProps(props, 'SliderOutput');
+
+  const { children: childrenProp, classNames: classNamesProp, ...rest } = props;
+
+  const theme = useTheme();
+
+  const classNames = useMemo(
+    () => mergeClassNames(sliderClassNames, theme.Slider, classNamesProp),
+    [theme.Slider, classNamesProp],
+  );
+
+  const style = useCallback(
+    (renderProps: SliderRenderProps) =>
+      inlineVars(sliderStateVars, renderProps),
+    [],
+  );
+
+  const values = useMemo<
+    [
+      [
+        typeof SliderOutputContext,
+        ContextValue<SliderOutputProps, HTMLOutputElement>,
+      ],
+    ]
+  >(() => [[SliderOutputContext, { classNames }]], [classNames]);
+
+  const children = useCallback(
+    (renderProps: SliderRenderProps) => (
+      <Provider values={values}>
+        {callRenderProps(childrenProp, renderProps)}
```

Comment:
You seem to have a container, but not a styleable element inside of the container